core/services/job/orm.go

```diff
@@ -364,6 +364,7 @@ func (o *orm) InsertJob(job *Job, qopts ...pg.QOpt) error {
 // DeleteJob removes a job
 func (o *orm) DeleteJob(id int32, qopts ...pg.QOpt) error {
 	o.lggr.Debugw("Deleting job", "jobID", id)
+	qopts = append(qopts, pg.WithOneMinuteTimeout())
```

Comment:
I think this is reasonable, although would be good to include a comment explaining why.

Also, while you are here, do you think you could fix this issue as well:

https://app.shortcut.com/chainlinklabs/story/32179/deleting-job-can-fail-and-leave-node-in-a-bad-state

Reply:
Sure, I'll add it as part of this PR.